packages/material-ui/src/CardHeader/CardHeader.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+
+declare var yada: () => void;
+
+const CardHeaderTest = () => {
+  // eslint-disable-next-line react/jsx-filename-extension
+  const CustomComponent: React.FC<{ prop1: string; prop2: number }> = () => <div />;
+  function notAnElementType(): void {
+    yada();
+  }
+  function MyComponent() {
+    return <div />;
+  }
+  return (
+    // eslint-disable-next-line react/jsx-filename-extension
```

Comment:
Can you remove all the unrelated noise from this test so that it is apparent what is being tested? This includes: export default, eslint-disable, `<Card>` etc.